src/main/java/at/hannibal2/skyhanni/utils/json/SkippingTypeAdapterFactory.kt

```diff
@@ -29,8 +29,9 @@ object SkippingTypeAdapterFactory : TypeAdapterFactory {
             return try {
                 parent.read(reader)
             } catch (e: Exception) {
-                // TODO include path and value found (as string)
-                SkyHanniMod.logger.warn("Failed to read value from JSON, skipping", e)
+                val foundValue = reader.peek().toString()
```

Comment:
what does peek return if !reader.hasnext()

Reply:
![image](https://github.com/user-attachments/assets/59b70a14-b06a-4710-8438-ddb785624ad5)

Nothing as in EOF? Seems like there's a case for that, unless I'm misunderstanding your question.

Reply:
Skipping reading JSON from failed path '$.players..profiles..sackContents.', found value of 'END_OBJECT'.